### ISSUE TITLE
Can't pipe output correctly

### ISSUE THREAD

**Issue by rawsyntax** [rentpath/rpenv#14]
This does not work:
```sh
rpenv ci|grep mysql
```

This does:
```sh
rpenv ci|&grep mysql
```

Just the pipe should work

**Comment by Christopher-Bui** [comment]
```
-rw-r-xr-x    1 root root     4.5M Mar  6  2015 rpenv
```

Looks like version 1.0.1 is on bot and I think 2.0.0 fixes this.

**Comment by colinrymer** [comment]
This was fixed by v2.0.0.
